juan_david_hernandez/ejercicio.md

```diff
@@ -0,0 +1,25 @@
+~~~
+Sub mientras()
+   monto = 0
+    While monto < 3000000
+        dinero = InputBox("Cuánto va a donar?")
+    
+        If dinero > 0 Then
+            aportaron = aportaron + 1
+            If dinero >= 10000 Then
+                aportaron10 = aportaron10 + 1
+            End If
+        Else
+            no_aportaron = no_aportaron + 1
+        End If
+        monto = monto + dinero
```

Comment:
This line must be inside true condition.

Reply:
I send the new code with the lines inside the condition